File: bialydunajec-academic-ministry/application/src/main/kotlin/org/bialydunajec/academicministry/application/dto/AcademicPriestDto.kt
package org.bialydunajec.academicministry.application.dto

import org.bialydunajec.ddd.application.base.dto.ExtendedDescriptionDto

data class AcademicPriestDto(
        val academicPriestId: String,
        val firstName: String,
        val lastName: String,
        val personalTitle: PersonalTitleDto?,
        val emailAddress: String?,
        val phoneNumber: String?,
        val description: ExtendedDescriptionDto?,
        val photoUrl: String?
)

File: frontend-main/projects/bialydunajec-main/src/app/academic-ministries-cottages/service/rest/dto/academic-priest.dto.ts
import {PersonalTitleDto} from './personal-title.dto';
import {ExtendedDescriptionDto} from '../../../../shared/dto/extended-description.dto';

export class AcademicPriestDto {
  academicPriestId: string;
  firstName: string;
  lastName: string;
  personalTitle?: PersonalTitleDto;
  emailAddress?: string;
  phoneNumber?: string;
  description?: ExtendedDescriptionDto;
  photoUrl?: string;
}
